README.md

```diff
@@ -198,6 +198,61 @@ Team Description Papers (TDPs) and Videos presented at the
 - **[Sina](https://drive.google.com/file/d/1nUhDa0mb71TBu2VJmea6NfFw-xc3RMO6)**
 - **[Speshari](https://drive.google.com/file/d/1DA7DTUwTPm_msuHFJStDBgv8KP3rkGWs)**
 
+#### 2023 - Bordeaux
+
+##### Lightweight
+
+- **[Aperture Robotics](https://drive.google.com/file/d/1SiGYok_ipUDLa4foxfPOh8DWkbOzyykB/view)**
+- **[Apex](https://drive.google.com/file/d/1ry9NL1WJNenm2RpwQSRVYsaONInA_I49/view)**
+- **[Bodenseeluchse](https://drive.google.com/file/d/1hz0UKwb6EAhLUjKS7NWXLa0YQvE3ab4j/view)** (`Outstanding Design Award`) (`Top Poster & Presentation`)
```

Comment:
@dbscoach @Adman @aquahika @MikeRCJ @javier687 @marcodankel Let me know if this is too much -- I just thought it would be best to highlight it in here as well